macros/edr/system/system_utils/get_config_var.sql

```diff
@@ -19,8 +19,6 @@
 {# We use this macro to define and call vars, as the global vars defined in dbt_project.yml
    of the package are not accessible at on-run-start and on-run-end #}
   {% set default_config = {
-    'detection_period': elementary.get_period_default_var(period='day', count=2),
-    'training_period': elementary.get_period_default_var(period='day', count=14),
```

Comment:
as these overwrite `days_back` and `backfill_days`, these default var values overwrite model/test values